Gems/Atom/Tools/ShaderManagementConsole/Code/Source/ShaderManagementConsoleApplication.h

```diff
@@ -13,13 +13,15 @@
 #include <AzToolsFramework/API/EditorWindowRequestBus.h>
 #include <ShaderManagementConsoleRequestBus.h>
 #include <Window/ShaderManagementConsoleWindow.h>
+#include <AzToolsFramework/AssetBrowser/AssetBrowserBus.h>
 
 namespace ShaderManagementConsole
 {
     class ShaderManagementConsoleApplication
         : public AtomToolsFramework::AtomToolsDocumentApplication
         , private ShaderManagementConsoleRequestBus::Handler
         , private AzToolsFramework::EditorWindowRequestBus::Handler
+        , public AzToolsFramework::AssetBrowser::AssetBrowserInteractionNotificationBus::Handler
```

Comment:
I don't see where the asset browser interaction notification bus is being used, except for where it's connected. Can this be Removed or reverted if it's not necessary anymore?

Reply:
Indeed it wasn't necessary (perhaps anymore). Removed it